proposals/range-mappings.md

```diff
@@ -82,17 +85,17 @@ Generate Line 1 Column 0 -> Original Line 2 Column 2 (range mapping)
 ```
 
 ``` js
-console.log("hello world");
-^       ^   ^
-|       |   + maps to Original Line 2 Column 14
+console.log(\n"hello world");
+^       ^     ^
+|       |     + maps to Original Line 3 Column 0
 |       + maps to Original Line 2 Column 10
 + maps to Original Line 2 Column 2
 ```
 
 ### Encoding
 
 To avoid a breaking change to the `mappings` field, a new field named `rangeMappings` is added.
-It contains encoded data per line in the generated code.
+It contains encoded data per-line in the generated code.
```

Comment:
We don't need to update this because the list of mappings is still per-line, right?

e.g. if we have a mapping at `0:0->0:0` and one at `2:0->3:0` (so on two different lines), to mark them as range mappings we would encode them as `A;;A` in `rangeMappings`, but the first one would span two lines because it spans up to the end of the mapping after?

Reply:
Correct.

This specific text change is purely a typo fix.  The encoding for `mappings` and `rangeMappings` remains per-line.

The encoding remains per-line and per-mapping.  So in your example, if you have two range mappings, you still need two entries in `rangeMappings`.

So I'm agreeing with you.  The extent of a range mapping is from the tagged mapping until the next mapping - regardless of whether that mapping is tagged as range mapping or not.